File: tasks/task12_1/point.py
from math import hypot

class Point:
    def __init__(self, x, y):
        self.x = x
        self.y = y

    def distance(self):
        return hypot(self.x, self.y)

    def distanceTo(self, p):
        return hypot(self.x - p.x, self.y - p.y)

x = float(input("Enter x coordinate: "))
y = float(input("Enter y coordinate: "))

p = Point(x, y)
p2 = Point(4.5, 7.0)
print(p.distance())
print(p.distanceTo(p2))

File: tasks/task12_1/Point.kt
import kotlin.math.hypot

class Point(var x: Double, var y: Double) {
    fun distance() = hypot(x, y)
    fun distanceTo(p: Point) = hypot(x - p.x, y - p.y)

}

fun main() {
    print("Enter x coordinate: ")
    val x = readln().toDouble()
    print("Enter y coordinate: ")
    val y = readln().toDouble()

    val p = Point(x, y)
    val p2 = Point(4.5, 7.0)
    println(p.distance())
    println(p.distanceTo(p2))
}